.github/workflows/_rust.yml

```diff
@@ -79,13 +79,30 @@ jobs:
     uses: ./.github/workflows/_rust_tests.yml
     with:
       isRust: ${{ inputs.isRust }}
+      consensusProtocol: "mysticeti"
       isPgIntegration: ${{ inputs.isPgIntegration }}
       isMoveExampleUsedByOthers: ${{ inputs.isMoveExampleUsedByOthers }}
       runSimtest: true
       testOnlyChangedCrates: true
       changedCrates: ${{ join(fromJson(needs.crates-changes.outputs.components || '[]'), ' ') }}
       changedExternalCrates: ${{ join(fromJson(needs.external-changes.outputs.components || '[]'), ' ') }}
 
+  #  rust-starfish-tests:
```

Comment:
What is this?

Reply:
A commented out action for running tests with Starfish for when we want to run it for every PR. Do you think we should get rid of it here and introduce it when we need it?